packages/mds-metrics/api.ts

```diff
@@ -45,6 +51,12 @@ function api(app: express.Express): express.Express {
     getEventCounts
   )
 
+  app.get(
+    pathsFor('/dump_metrics'),
```

Comment:
/metrics/all   ??

Reply:
This would just translate to `app.get(pathsFor('/all')...`